src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double secondLeg, double firstLeg) {
+        super(color);
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }
+
+    public double getFirstLeg() {
+        return firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void moreInfo() {

```

Comment:
The method name 'moreInfo' should be a verb or a verb phrase, reflecting the action it performs.